testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/JdkModuleDependencyTestCase.java

```diff
@@ -0,0 +1,214 @@
+package org.wildfly.core.test.standalone.mgmt.api.core;
```

Comment:
This class is missing a copyright header.

Reply:
Fixed.